tenable/io/v3/vm/agent_config.py

```diff
@@ -0,0 +1,93 @@
+'''
+Agent Config
+============
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`agent config <agent-config>` API endpoints.
+
+Methods available on ``tio.agent_config``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AgentConfigAPI
+    :members:
+'''
+from typing import Dict
+
+from ..base.endpoints.explore import ExploreBaseEndpoint
+
+
+class AgentConfigAPI(ExploreBaseEndpoint):
+    '''
+    This will contain all methods related to agent config
+    '''
+    # _path: str = 'api/v3/agents'
```

Comment:
Please update the path as per it will be expected in v3.

Reply:
Done.